web-console/src/lib/functions/common/tuple.ts

```diff
@@ -0,0 +1,7 @@
+export function tuple<T extends unknown[]>(...t: [...T]) {
```

Comment:
maybe would be helpful to add a doc comment and an example how to use for future users